presto-main/src/main/java/io/prestosql/metadata/SignatureBinder.java

```diff
@@ -855,6 +855,14 @@ public SolverReturnStatus update(BoundVariables bindings)
                             return SolverReturnStatus.UNSOLVABLE;
                         }
                         TypeSignature typeSignature = type.get().getTypeSignature();
+                        verify(
+                                i < typeSignature.getParameters().size(),
+                                "Expected type signature %s of type %s (coerced %s to %s base name) to have size greater than %s",
```

Comment:
may be replace `size` with `parameter count` in the error message?

Reply:
thanks